.gitattributes

```diff
@@ -0,0 +1,32 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* eol=lf
```

Comment:
This seems to be an old template
How did you generate this?

Reply:
Can you help me understand where this template comes from? So we can fix the old sources for it?
The other files might have similar issues btw.

Reply:
I took this file from a fresh module from the core. As I see, all the `.gitattributes` files are the same except some outdated ones. Unfortunately, for the first time, I took an outdated one.

Reply:
Please always use the tools to create those files for now, we should never copy from existing modules.